mod-kb-ebsco-java/mod-kb-ebsco-java.postman_collection.json

```diff
@@ -13104,7 +13104,7 @@
 												],
 												"body": {
 													"mode": "raw",
-													"raw": " {\n  \"data\": {\n    \"type\": \"resources\",\n    \"attributes\": {\n      \"isPeerReviewed\": false,\n      \"name\": \"custom title-{{custom-resource-uuid}}\",\n      \"publicationType\": \"Book\",\n      \"isSelected\": true,\n      \"visibilityData\": {\n        \"isHidden\" : false\n      },\n      \"customEmbargoPeriod\": {\n      \"embargoUnit\" : \"Months\",\n      \"embargoValue\" : 5\n      },\n      \"customCoverages\" : [\n      {\n      \"beginCoverage\" : \"2001-01-01\",\n      \"endCoverage\" : \"2004-02-01\"\n      }\n      ],\n      \"coverageStatement\": \"Test Coverage Statement\",\n      \"proxy\" : {\n      \"id\" : \"<n>\"\n      }\n    }\n  }\n}"
+													"raw": "{\r\n  \"data\": {\r\n    \"id\": \"{{custom-resourceid}}\",\r\n    \"type\": \"resources\",\r\n    \"attributes\": {\r\n      \"packageId\": \"{{custom-packageid-for-resource}}\",\r\n      \"titleId\": \"{{custom-titleid-for-resource}}\",\r\n      \"url\": \"http://test\",\r\n      \"name\": \"custom title-{{custom-resource-uuid}}\",\r\n      \"isPeerReviewed\": false,\r\n      \"publicationType\": \"Book\",\r\n      \"providerId\": \"{{custom-providerid}}\",\r\n      \"providerName\": \"API DEV GOVERNMENT CUSTOMER\",\r\n      \"packageName\": \"custom-package-{{custom-package-for-resource-uuid}}\",\r\n      \"isSelected\": true,\r\n      \"managedCoverages\": [],\r\n      \"managedEmbargoPeriod\": {\r\n        \"embargoValue\": 0\r\n      },\r\n      \"visibilityData\": {\r\n        \"isHidden\": false\r\n      },\r\n      \"customEmbargoPeriod\": {\r\n        \"embargoUnit\": \"Months\",\r\n        \"embargoValue\": 5\r\n      },\r\n      \"customCoverages\": [\r\n        {\r\n          \"beginCoverage\": \"2001-01-01\",\r\n          \"endCoverage\": \"2004-02-01\"\r\n        }\r\n      ],\r\n      \"coverageStatement\": \"Test Coverage Statement\",\r\n      \"proxy\": {\r\n        \"id\": \"<n>\"\r\n      },\r\n      \"publisherName\": \"test publisher\",\r\n      \"edition\": \"test edition\",\r\n      \"publicationType\": \"Database\",\r\n      \"description\": \"test description\",\r\n      \"subjects\": [],\r\n      \"contributors\": [\r\n        {\r\n          \"type\": \"author\",\r\n          \"contributor\": \"smith, john\"\r\n        },\r\n        {\r\n          \"type\": \"illustrator\",\r\n          \"contributor\": \"smith, ralph\"\r\n        }\r\n      ],\r\n      \"identifiers\": [\r\n        {\r\n          \"id\": \"11-2222-3333\",\r\n          \"type\": \"ISSN\",\r\n          \"subtype\": \"Online\"\r\n        }\r\n      ],\r\n      \"isTitleCustom\": true\r\n    }\r\n  }\r\n}"
```

Comment:
somehow, a carriage return got appended to all your changes @andrii-paias - could you please take care of that before merging? Thanks!